### DISCUSSION TITLE
How to collect metrics for LIF, home node etc.?

### DISCUSSION THREAD

**Discussion by jmg011** [NetApp/harvest#988] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Metrics for LIFs

**Describe alternatives you've considered**
Custom Monitor

**Additional context**
N/A

**Comment by ruanruijuan** [comment, marked as answer]
@jmg011 you can use the net-interface-get-iter.net-interface-info to get the LIF related metrics. For the home node specifically, that's the attribute net-interface-get-iter.net-interface-info.current-node.

**Comment by jmg011** [comment]
Thank you!

**Comment by ruanruijuan** [comment]
Question was answered. Closing.
